salt/minion.sls

```diff
@@ -104,8 +106,7 @@ salt-minion:
     - enable: {{ salt_settings.minion_service_details.enabled }}
     - name: {{ salt_settings.minion_service }}
     - watch:
-      - file: salt-master-macos
-      - file: salt-master
+      # file: salt-master
```

Comment:
I would not comment it out, I would remove the line if it's not needed.

Reply:
yes, thanks.